com.vrcfury.vrcfury/Runtime/VF/Model/Feature.cs

```diff
@@ -249,6 +249,7 @@ public class Toggle : LegacyFeatureModel2 {
         [NonSerialized] public bool usePrefixOnParam = true;
         [NonSerialized] public string paramOverride = null;
         [NonSerialized] public bool useInt = false;
+        public bool hasExitTime = false;

```

Comment:
Is there supposed to be an editor for this?

Reply:
you told me not to add one, its either 0 or 1

Reply:
It doesn't look like it's possible for this ever be true now though?